### PR TITLE
bat: edit page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by fazlearefin** [comment]
This page look to be quite old having some not so useful examples like appending files. I have added more useful and exciting examples as bat has matured over the years.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/bat.md [hunk 1]
```diff
@@ -4,26 +4,34 @@
...
 
 - Concatenate several files into the target file:
 
 `bat {{path/to/file1 path/to/file2 ...}} > {{path/to/target_file}}`
 
-- Append several files into the target file:
+- Remove decorations and auto paging:
 
-`bat {{path/to/file1 path/to/file2 ...}} >> {{path/to/target_file}}`
+`bat {{--style=plain --pager=never|-pp}} {{path/to/file}}`
```
Comment (gutjuri) [inline pages/common/bat.md:17]:
```suggestion
`bat --style=plain --pager={{never|-pp}} {{path/to/file}}`
```

File: pages/common/bat.md [hunk 2]
```diff
@@ -4,26 +4,34 @@
...
 
-- Number all output lines:
+- Highlight specific line or range of lines with different background color:
```
Comment (gutjuri) [inline pages/common/bat.md:19]:
```suggestion
- Highlight a specific line or a range of lines with a different background color:
```

File: pages/common/bat.md [hunk 3]
```diff
@@ -4,26 +4,34 @@
...
-`bat --number {{path/to/file}}`
+`bat {{--highlight-line|-H}} {{10|5:10|:10|10:|10:+5}} {{path/to/file}}`
 
-- Syntax highlight a JSON file:
+- Show non-printable characters like space, tab or newline:
 
-`bat --language json {{path/to/file.json}}`
+`bat {{--show-all|-A}} {{path/to/file}}`
+
+- Remove decorations except line numbers in output:
```
Comment (gutjuri) [inline pages/common/bat.md:27]:
```suggestion
- Remove decorations except line numbers in the output:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have some comments

File: pages/common/bat.md (hunk 1 above)
Comment (acuteenvy) [reply pages/common/bat.md:17]: The fact that this suggestion was made means that a potential user may also understand `|-pp` as an argument to `--pager`. How about mentioning it in the description instead?

File: pages/common/bat.md [hunk 4]
```diff
@@ -4,26 +4,34 @@
...
+- Pretty print the contents of one or more files to `stdout`:
 
 `bat {{path/to/file1 path/to/file2 ...}}`
 
 - Concatenate several files into the target file:
 
 `bat {{path/to/file1 path/to/file2 ...}} > {{path/to/target_file}}`
 
-- Append several files into the target file:
+- Remove decorations and auto paging:
```
Comment (acuteenvy) [inline pages/common/bat.md:15]: [See my comment above](https://github.com/tldr-pages/tldr/pull/12451/files#r1511314239)
```suggestion
- Remove decorations and disable paging (`--style plain` can be replaced with `-p`, or both options with `-pp`):
```

File: pages/common/bat.md (hunk 1 above)
Comment (acuteenvy) [inline pages/common/bat.md:17]:
```suggestion
`bat --style=plain --pager=never {{path/to/file}}`
```

**Comment by fazlearefin** [comment]
@acuteenvy Opted for the second option (of not having `-pp` at all) as it is much easier for the user to follow.

File: pages/common/bat.md [hunk 5]
```diff
@@ -4,26 +4,34 @@
...
 
-`bat {{path/to/file1 path/to/file2 ...}} >> {{path/to/target_file}}`
+`bat --style=plain --pager=never {{path/to/file}}`
```
Comment (acuteenvy) [inline pages/common/bat.md:17]:
```suggestion
`bat --style plain --pager never {{path/to/file}}`
```

**Review by acuteenvy — COMMENTED** [review]
What's hard to follow in that description?

File: pages/common/bat.md [hunk 6]
```diff
@@ -4,26 +4,34 @@
...
-`bat --number {{path/to/file}}`
+`bat {{--highlight-line|-H}} {{10|5:10|:10|10:|10:+5}} {{path/to/file}}`
 
-- Syntax highlight a JSON file:
+- Show non-printable characters like space, tab or newline:
 
-`bat --language json {{path/to/file.json}}`
+`bat {{--show-all|-A}} {{path/to/file}}`
+
+- Remove decorations except line numbers in the output:
```
Comment (acuteenvy) [inline pages/common/bat.md:27]:
```suggestion
- Remove all decorations except line numbers in the output:
```

**Review by acuteenvy — APPROVED** [review]
